models/eprop_iaf_psc_delta.h

```diff
@@ -35,72 +35,122 @@
 
 namespace nest
 {
-
+// Disable clang-formatting for documentation due to over-wide table.
+// clang-format off
 /* BeginUserDocs: neuron, e-prop plasticity, integrate-and-fire, current-based
 
 Short description
 +++++++++++++++++
 
-Current-based leaky integrate-and-fire neuron model with delta-shaped
-postsynaptic currents for e-prop plasticity
+Leaky integrate-and-fire model with delta-shaped input currents
+for e-prop plasticity
 
 Description
 +++++++++++
 
-``eprop_iaf_psc_delta`` is an implementation of a leaky integrate-and-fire model
-where the potential jumps on each spike arrival used for eligibility propagation
-(e-prop) plasticity.
+``iaf_psc_delta`` is a leaky integrate-and-fire neuron model with
+
+* a hard threshold,
+* a fixed refractory period,
+* Dirac delta (:math:`\delta`)-shaped synaptic input currents.
+* support for eligibility propagation (e-prop).
 
 It is the standard ``iaf_psc_delta`` model endowed with e-prop plasticity.
 
-The threshold crossing is followed by an absolute refractory period
-during which the membrane potential is clamped to the resting potential.
+Membrane potential evolution, spike emission, and refractoriness
+................................................................
+
+The membrane potential evolves according to
+
+.. math::
+
+   \frac{dV_\text{m}}{dt} = -\frac{V_{\text{m}} - E_\text{L}}{\tau_{\text{m}}} + \dot{\Delta}_{\text{syn}} + \frac{I_{\text{syn}} + I_\text{e}}{C_{\text{m}}}
+
+where the derivative of change in voltage due to synaptic input :math:`\dot{\Delta}_{\text{syn}}(t)` is discussed below and :math:`I_\text{e}` is
+a constant input current set as a model parameter.
+
+A spike is emitted at time step :math:`t^*=t_{k+1}` if
+
+.. math::
+
+   V_\text{m}(t_k) < V_{th} \quad\text{and}\quad V_\text{m}(t_{k+1})\geq V_\text{th} \;.
+
+Subsequently,
+
+.. math::
+
+   V_\text{m}(t) = V_{\text{reset}} \quad\text{for}\quad t^* \leq t < t^* + t_{\text{ref}} \;,
```

Comment:
```suggestion
A spike is emitted at time step :math:`t` if

.. math::

   V_\text{m}(t) < V_{th} \quad\text{and}\quad V_\text{m}(t+1)\geq V_\text{th} \;.

Subsequently,

.. math::

   V_\text{m}(t') = V_{\text{reset}} \quad\text{for}\quad t' \leq t < t' + t_{\text{ref}} \;,
```
It seems to me that `t_{k+1}` does not necessarily conveys that `t_{k+1} = t_{k} + 1`. On the other hand, the label $k$ is used as subscript here while used as superscript a in the following section.

Reply:
I would not change this part of the user documentation since it is exactly the same as the `iaf_psc_delta`. Apparently, since we copied the model there have been some changes to it upstream. We need to check again after we merge the current master if there has not been some more.

Reply:
Maybe it helps for the review to compare the diff between `iaf_psc_delta.{cpp,h}` with `eprop_iaf_psc_delta.{cpp,h}.